filters/auth/filters.go

```diff
@@ -47,7 +49,21 @@ func (authFilter AuthenticationFilter) Filters() []web.Filter {
 			RouteMatcher: web.RouteMatcher{
 				PathPattern: "/v1/service_brokers/**",
 			},
-			Middleware: authFilter.basicAuth,
+			Middleware: authFilter.filterDispatcher,
```

Comment:
Currently, we allow with platform credentials to delete, create and update brokers. This should not be the case.